src/physics/utils/orbital_data.meta

```diff
@@ -0,0 +1,21 @@
+[ccpp-table-properties]
+  name = orbital_data
+  type = module
+[ccpp-arg-table]
+  name = orbital_data
+  type = module
+[solar_declination]
+  standard_name = solar_declination
+  units = radians
```

Comment:
in keeping with the SI units we (recently) decided on, this (and `solar_zenith_angle`) should be "rad".

if you want to just include this update in your existing PR - https://github.com/ESCOMP/atmospheric_physics/pull/181 that would be swell.

Reply:
will do!